homeassistant/components/loopenergy/sensor.py

```diff
@@ -113,7 +113,8 @@ def unit_of_measurement(self):
         return self._unit_of_measurement
 
     def _callback(self):
-        self.schedule_update_ha_state(True)
+        if self.hass is not None:
+            self.schedule_update_ha_state(True)
```

Comment:
The correct fix will be to attach the callback inside `async_added_to_hass`:
https://developers.home-assistant.io/docs/core/entity/#lifecycle-hooks

Reply:
Excellent - thanks - now done.